### ISSUE TITLE
Docsify g command generate unwanted structure

### ISSUE THREAD

**Issue by yzqdev** [docsifyjs/docsify-cli#173]
> it's on windows

My file struct is:
```text
└─doctest
        .nojekyll
        index.html
        markdown.md
        more-pages.md
        my404.md
        plugins.md
        pwa.md
        quickstart.md
        README.md
        ssr.md
        SUMMARY.md
        themes.md
        vue.md
        write-a-plugin.md
        _sidebar.md
```

after i run `docsify g docstest`, it generates absolute path.
the generated sidebar is :
```md
- E:
  - [markdown](E:\tmp\docs\doctest\markdown.md)
  - [more-pages](E:\tmp\docs\doctest\more-pages.md)
  - [my404](E:\tmp\docs\doctest\my404.md)
  - [plugins](E:\tmp\docs\doctest\plugins.md)
  - [pwa](E:\tmp\docs\doctest\pwa.md)
  - [quickstart](E:\tmp\docs\doctest\quickstart.md)
  - [README](E:\tmp\docs\doctest\README.md)
  - [ssr](E:\tmp\docs\doctest\ssr.md)
  - [SUMMARY](E:\tmp\docs\doctest\SUMMARY.md)
  - [themes](E:\tmp\docs\doctest\themes.md)
  - [vue](E:\tmp\docs\doctest\vue.md)
  - [write-a-plugin](E:\tmp\docs\doctest\write-a-plugin.md)
```
what i want is relative path like this:
```text
- E:
  - [markdown](./markdown.md)
  - [more-pages](./more-pages.md)
  - [my404](./my404.md)
  - [plugins](./plugins.md)
  - [pwa](./pwa.md)
  - [quickstart](./quickstart.md)
  - [README](./README.md)
  - [ssr](./ssr.md)
  - [SUMMARY](./SUMMARY.md)
  - [themes](./themes.md)
  - [vue](./vue.md)
  - [write-a-plugin](./write-a-plugin.md)
```
so how should i config ?

**Comment by sy-records** [comment]
There may be a problem with windows, I will check it later.

**Comment by sy-records** [comment]
Fixed via #192
